onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -1036,8 +1036,30 @@ Applies to session load, initialization, etc. Default is 0.)pbdoc")
                                 onnxruntime::FreeDimensionOverrideType::Name,
                                 dim_value}); },
           "Rpbdoc(Specify values of named dimensions within model inputs.)pbdoc")
-      .def_readwrite("use_prepacking", &SessionOptions::use_prepacking,
-                     R"pbdoc(Whether to use prepacking. Default is true.)pbdoc");
+      .def(
+          "add_session_config_entry",
+          [](SessionOptions* options, const char* config_key, const char* config_value) -> void {
+            std::string key(config_key);
```

Comment:
Instead of duplicating validation here and in the C API, can there be a method in SessionOptions (or a shared helper) for adding a config key that is called from both places?